woocommerce/woocommerce-gutenberg-products-block
# E2E: Ensure the Shop page is created

## What

This PR fixes the environment setup script issue after #10611 by creating the shop page after emptying the site.

## Why

In the original setup script, we don't create the shop page after empty the site, but because we run script in sequence, Woo has time to generate it. In #10611, we optimized the setup to break down the script and run smaller sets in parallel. This results in, sometimes, the Shop page isn't created, which cause 404 error when the tests navigate to `/shop/`, thus makes the tests failed.



## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

Ensure the Playwright tests are passed on both CI and local.


## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Add suggested changelog entry here.
